test/testgroups/cluster_crd.go

```diff
@@ -3,50 +3,15 @@ package testgroups
 import (
 	"testing"
 
-	"github.com/tektoncd/operator/test/config"
-	"github.com/tektoncd/operator/test/helpers"
-	"github.com/tektoncd/operator/test/testsuites"
-
 	"github.com/operator-framework/operator-sdk/pkg/test"
-	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+	"github.com/tektoncd/operator/test/testsuites"
 )
 
 // ClusterCRD is the test group for testing config.operator.tekton.dev CRD
 func ClusterCRD(t *testing.T) {
 	ctx := test.NewTestCtx(t)
 	defer ctx.Cleanup()
 
-	err := deployOperator(t, ctx)
-	helpers.AssertNoError(t, err)
-
 	t.Run("auto-installs-pipelines", testsuites.ValidateAutoInstall)
 	t.Run("delete-pipelines", testsuites.ValidateDeletion)
 }
-
-func deployOperator(t *testing.T, ctx *test.TestCtx) error {
```

Comment:
Why is this removed ?

Reply:
function tekton_setup() in the script will launch the operator, so we do not need to deploy the operator here.